### PR TITLE
add/edit round done

### PR DESCRIPTION
**Pull request by SamuelOmodele**

### CODE REVIEW

**Comment by a-k-o-r-e-d-e** [comment]
@SamuelOmodele can you please resolve the conflicts so i can merge this in?

**Comment by SamuelOmodele** [comment]
> @SamuelOmodele can you please resolve the conflicts so i can merge this in?

I've resolved it.

**Comment by a-k-o-r-e-d-e** [comment]
I can see a conflict

**Comment by a-k-o-r-e-d-e** [comment]
> > @SamuelOmodele can you please resolve the conflicts so i can merge this in?
> 
> I've resolved it.

i can still see a conflict in the EditQuix.tsx file

File: src/Pages/Round/AddEditRound.tsx
Comment (a-k-o-r-e-d-e) [inline src/Pages/Round/AddEditRound.tsx:1]: date could have been formatted using the date-fns package but we will allow

**Review by a-k-o-r-e-d-e — CHANGES_REQUESTED** [review]

File: src/Pages/Round/AddEditRound.tsx
Comment (SamuelOmodele) [reply src/Pages/Round/AddEditRound.tsx:1]: Okay sir. 

I have resolved the conflicts. I saw the same conflict when i first created the Pull request and corrected it then, I don't know why it came up again.

**Comment by SamuelOmodele** [comment]
> > > @SamuelOmodele can you please resolve the conflicts so i can merge this in?
> > 
> > 
> > I've resolved it.
> 
> i can still see a conflict in the EditQuix.tsx file

I have corrected it

**Review by a-k-o-r-e-d-e — APPROVED** [review]

**Pull request merged**